docs/src/pages/components/data-grid/rendering/rendering.md

```diff
@@ -207,6 +207,22 @@ This means that the number of rows will drive the height of the grid and consequ
 
 {{"demo": "pages/components/data-grid/rendering/AutoHeightGrid.js", "bg": "inline"}}
 
+## Size
+
+You can change the size of the rows and the column header.
+
+### Size prop
+
+The vertical size of the data grid can be set using the `size` prop. Regardless of its setting, the `size` prop will respect values provided using the `rowHeight` or `headerHeight` props.
+
+{{"demo": "pages/components/data-grid/rendering/SizePickerSmallGrid.js", "bg": "inline"}}
+
+### Size picker button
```

Comment:
There is an issue with the experimental feature. The documented demo doesn't work. The toolbar is hidden by default. This is related to https://github.com/mui-org/material-ui-x/pull/606#issuecomment-737130908.

Reply:
Ok, yes I forgot. I enabled the toolbar for the demo